### ISSUE TITLE
give more detailed information where to find launchers config file and what should be checked

### ISSUE THREAD

**Issue by aisbergde** [MicrosoftDocs/sql-docs#5916]
> a. Review the launcher's .config file and ensure that the working directory is valid.

These are very poor instructions. 

Provide instructions, how to locate "the launcher's .config file"
For example:
- look at the service properties for the "path to exe file"
- open the folder of this file
- search "pythonlauncher.config"

I hope, this is the right way to find the config file

Add some instructions what and how should be checked in this config file. I found some instructions here 
https://samtran.me/2019/01/05/sql-python-integration-problems/
for example the service user should be granted some rights in some folders in this config file. And some other issues related to some environment settings.

I think it should be possible to find a solution here in the microsoft documentation. And I suggest to add some instructions here:

- what should be checked in the config file
- which rights are required

It is very very very hard to troubleshoot Launchpad issues, and I am not able to solve issues for some hours.
If the setup is not able to cofigure the required rights in the "normal" setup process, then at least the troubleshooting should work.

First it is hard to find this troubleshooting site because there is no link from the installing instructions to the troubelshoot site here. There we can find only the result which should happen if no issues happen.

Then it is very hard to follow the instructions here. It is a lot of text, but it is hard to find the relevant information which could help to solve the issue.

My issues are these (on mssql 2017 and mssql 2019, both on the same machine)

> Unable to communicate with the runtime for 'Python' script for request id: 6DA07BF7-7FC1-4729-9204-1763515001A3. Please check the requirements of 'Python' runtime.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 61f56b22-f0e4-f98d-02d4-f4d605f13cee
* Version Independent ID: a2a14494-a69a-e292-e65b-a3490ecf16eb
* Content: [Troubleshoot issues with Launchpad service - SQL Server Machine Learning Services](https://docs.microsoft.com/en-us/sql/machine-learning/troubleshooting/common-issues-external-script-execution?view=sql-server-ver15)
* Content Source: [docs/machine-learning/troubleshooting/common-issues-external-script-execution.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/machine-learning/troubleshooting/common-issues-external-script-execution.md)
* Product: **sql**
* Technology: **machine-learning-services**
* GitHub Login: @dphansen
* Microsoft Alias: **davidph**

**Comment by WilliamAntonRohm** [comment]
@aisbergde -- Germo, thank you again for your feedback and details.

@dphansen -- David, please look into this issue.

**Comment by dphansen** [comment]
@aisbergde Thank you for your feedback. I will have a look at the issue and get back to you.

**Comment by dphansen** [comment]
@aisbergde We will update the documentation to include your suggestions and in general update them for clarification. Thank you for bringing this to our attention. For your specific issue, I will recommend that you [open a support case](https://support.serviceshub.microsoft.com/supportforbusiness) with Microsoft, and our support engineers will be able to help you.
